File: src/leetcode/python/NimGame_292.py
class NimGame_292(object):
    def canWinNim(self, n):
        """
        :type n: int
        :rtype: bool

        if n % 4 != 0, the first just take numbers that equals multiple of 4
        else lose, for the latter pick can do the same thing to win
        """
        return True if n % 4 != 0 else False

File: src/leetcode/java/NimGame_292.java
package leetcode.java.easy;

public class NimGame_292 {
  // if n % 4 != 0, the first just take numbers that equals multiple of 4
  // else lose, for the latter pick can do the same thing to win
  public boolean canWinNim(int n) {
    if (n % 4 == 0) return false;
    return true;
  }
}
